easybuild/easyconfigs/s/SeaView/SeaView-5.0.5-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'Tarball'
```

Comment:
So, rather than relying on binary installs, especially considering you have already added all the deps and moved this from SYSTEM to GCCcore, it just makes sense to build this from source, which would also add the benefit that this no longer just support x86